apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-06-24T14:47:44Z"
-  build_hash: 9c35918f8a98b41ada10cb3aa460dbf463428463
-  go_version: go1.16.4 linux/amd64
-  version: v0.2.3
-api_directory_checksum: 1c67711ffbdd9a4ed8e2051fbb70e84c7fd98b82
+  build_date: "2021-08-17T13:53:27Z"
+  build_hash: 64436ce7e8d7f7356daa6888600a68da4f09a5ac
+  go_version: go1.15.5 linux/amd64
+  version: v0.11.0
+api_directory_checksum: 3c829828db77ef587929da78f41d52b2459fb054
 api_version: v1alpha1
-aws_sdk_go_version: v1.38.60
+aws_sdk_go_version: v1.38.35
```

Comment:
Thoughts: Probably there no difference between `1.38.35` and `1.38.60` for ECR. But ideally the code generator should be able to pick up the latest used aws-sdk-go version from the `ack-generate-metadata.yaml` and reuse it as default version.